lib/web/sessions.go

```diff
@@ -65,18 +65,19 @@ func (c *SessionContext) getTerminal(sessionID session.ID) (*terminalHandler, er
 }
 
 func (c *SessionContext) UpdateSessionTerminal(
-	namespace string, sessionID session.ID, params session.TerminalParams) error {
+	siteAPI auth.ClientI, namespace string, sessionID session.ID, params session.TerminalParams) error {
 
-	err := c.clt.UpdateSession(session.UpdateRequest{
+	err := siteAPI.UpdateSession(session.UpdateRequest{
 		ID:             sessionID,
 		TerminalParams: &params,
 		Namespace:      namespace,
 	})
 	if err != nil {
-		return trace.Wrap(err)
+		log.Error(err)
```

Comment:
I think I understand why fallthrough is necessary, but can you add comment for future readers?